### ISSUE TITLE
Time step select in map

### ISSUE THREAD

**Issue by paulgirard** [Universite-Gustave-Eiffel/lasso#11]
Time ca be chnaged by clikcing on the timeline but it's not clear enough we should add a time selector underneath the map selector.

**Comment by pierromond** [comment]
- toggle activate time (sometimes impossible to put on)
- than add three time selector underneath map selector (season, day, hour)

**Comment by pierromond** [comment]
- if toggle is off, gray or remove time circles.
- toggle can be on top and bottom

**Comment by pierromond** [comment]
if timestampp is never null, the toggle can't be switched off and the first time selector is select

**Comment by paulgirard** [comment]
- un seul sélecteur recherchable 
- ajouter une option désactivé qui utilise le label de la couche temps moyen

**Comment by sim51** [comment]
I added a new button on the map to display the map selector :

![image](https://github.com/Universite-Gustave-Eiffel/lasso/assets/231657/adb16c90-ab44-4b1d-a321-f558007bb219)

When nothing is selected, the button is in black/white. 
When there is a time selection, the button has a "warning color" , the button's title has the current selection.
I also repeated the "selected" color on the donut to make the link between them. 

![image](https://github.com/Universite-Gustave-Eiffel/lasso/assets/231657/104f1ff6-ac8f-4a9f-88aa-a52a2d0e7209)

When you click on the button, a popin is displayed with an overlay on the current map : 

![image](https://github.com/Universite-Gustave-Eiffel/lasso/assets/231657/f7b099e3-c27c-459e-bcf0-711a4c8b34de)
 
@pierromond is it ok for you ?
